From 8d8d669c55ddd2bfe6847e1a6a1ba39368bf09e4 Mon Sep 17 00:00:00 2001
From: Kirill Kapranov <kirill.kapranov@compulab.co.il>
Date: Mon, 29 Jun 2020 18:15:26 +0300
Subject: [PATCH 43/49] Move environment from 4M to 17k address

Change starting address of environment data on eMMC form (64 * 64k) to (34 * 512)
in order to avoid possible conflicts with the first partition.

Signed-off-by: Kirill Kapranov <kirill.kapranov@compulab.co.il>
---
 include/configs/ucm-imx8m-mini.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/ucm-imx8m-mini.h b/include/configs/ucm-imx8m-mini.h
index c153f61ea9..b902d5eb04 100644
--- a/include/configs/ucm-imx8m-mini.h
+++ b/include/configs/ucm-imx8m-mini.h
@@ -197,7 +197,7 @@
         (CONFIG_SYS_INIT_RAM_ADDR + CONFIG_SYS_INIT_SP_OFFSET)
 
 #define CONFIG_ENV_OVERWRITE
-#define CONFIG_ENV_OFFSET               (64 * SZ_64K)
+#define CONFIG_ENV_OFFSET               (34 * 512) /* Max possible GPT size */
 #define CONFIG_ENV_SIZE			0x1000
 #define CONFIG_SYS_MMC_ENV_DEV		0   /* USDHC2 */
 #define CONFIG_MMCROOT			"/dev/mmcblk1p2"  /* USDHC2 */
-- 
2.11.0

